ws-server/src/server/module.rs

```diff
@@ -122,6 +122,66 @@ impl RpcModule {
 		Ok(SubscriptionSink { method: subscribe_method_name, subscribers })
 	}
 
+	/// Register a new RPC subscription, with subscribe and unsubscribe methods.
+	pub fn register_subscription_with_params<P: DeserializeOwned + Send + Sync + 'static>(
```

Comment:
todo: remove boiler plate when we decided whether this is ok or not

Reply:
Do we need two methods for this? Wouldn't it be enough to have a default `()` here?

Reply:
yeah, good point completely useless